### DISCUSSION TITLE
Can anyone Tell me how to use this ,

### DISCUSSION THREAD

**Discussion by YaswanthDasamandam** [10cl/chatdev#17] [Q&A]
I am confused How to use this can anyone tell me how to use this . Is there any starter guide for this

**Comment by 10cl** [comment]
This is an extension that requires your browser to log in to the ChatGPT website or other LLMs  then use it through ChatGPT.
https://github.com/10cl/chatdev#-chat-mode
1. choose your LLM 2. Use the keyboard to control the player's actions,  3. find npc, then enter the box to start chatting.

and I'll provide a video presentation tomorrow.

**Comment by YaswanthDasamandam** [comment]
I see that , here It is building , I am unable to figure out How to change the structure of process here , 
I asked it to build a website from the build website gpt , Are there any other GPT's , So I can use it to create projects

**Comment by YaswanthDasamandam** [comment]
Also where is the project stored

**Comment by 10cl** [comment]
The current GPTs is just creating a html website and has no storage yet, other GPTs are on the way, and I'm actively adding more GPTs and tutorials.  if you want to create projects & store it,  I need to think more about this GPTs, or the probability of failure will waste too much time

**Reply by YaswanthDasamandam** [reply]
Can you explain how it works ,I will try to contribute

**Reply by 10cl** [reply]
Great, you can see the introduction of PromptIDE, open the edit of Gpts,  click `func` ，is the regular processing of the whole reply, get content for ```html xxx，the processed variable as the local variable of this node
